Vagrantfile

```diff
@@ -1,37 +1,122 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
-require 'yaml'
+require "yaml"
 
-def load_security
-  fname = File.join(File.dirname(__FILE__), "security.yml")
-  if !File.exist? fname
-    $stderr.puts "security.yml not found - please run `./security-setup` and try again."
-    exit 1
-  end
+# Number of VMs to create
+WORKERS=1
```

Comment:
could we add support for optionally reading these variables (and other variables like cpu/memory) from an external config file that can be gitignored? that way you can maintain a custom vagrant setup without a dirty working tree